src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.RemoteServer.cs

```diff
@@ -342,14 +339,23 @@ public async Task SendAsync_HttpRequestMsgResponseHeadersRead_StatusCodeOK(Confi
 
         [OuterLoop("Uses external servers", typeof(PlatformDetection), nameof(PlatformDetection.LocalEchoServerIsNotAvailable))]
         [Theory, MemberData(nameof(RemoteServersMemberData))]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/37669", TestPlatforms.Browser)]
         public async Task PostAsync_CallMethodTwice_StringContent(Configuration.Http.RemoteServer remoteServer)
         {
             using (HttpClient client = CreateHttpClientForRemoteServer(remoteServer))
             {
                 string data = "Test String";
                 var content = new StringContent(data, Encoding.UTF8);
-                content.Headers.ContentMD5 = TestHelper.ComputeMD5Hash(data);
+
+                if(PlatformDetection.IsBrowser)
```

Comment:
NIT (same as other places)
```suggestion
                if (PlatformDetection.IsBrowser)
```